### PR TITLE
Bug 1219787 - Imagecompare RTL: wait until APN list is populated. 

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4af2061fd3f82d328fea0894c8300f60708355f7

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py [hunk 1]
```diff
@@ -12,8 +12,8 @@ def test_settings_app(self):
         settings = Settings(self.marionette)
         settings.launch()
 
-        # opening each subpage in Settings menu.
-        # some subpages have their own subpages, and they need to be opened as well.
+        # # opening each subpage in Settings menu.
+        # # some subpages have their own subpages, and they need to be opened as well.
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py]: This seems like an unneeded/unwanted change.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=06953eba0a0eba4bc4b16ddf22dda7003191f403

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=17543add5467dda07fe9e9c6790096d2bc7c58fe

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py [hunk 2]
```diff
@@ -108,8 +108,8 @@ def test_settings_app(self):
         settings.return_to_prev_menu(cs_page.cs_screen_element, cs_page.call_forwarding_screen_element)
 
         cs_page.tap_call_barring()
-        self.take_screenshot('callBarring-init')
-        cs_page.wait_until_call_barring_info_received()
+        cs_page.wait_until_call_barring_info_received()  # this sometimes happens in less than a second,
+                                                         # making it difficult to obtain init view
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py]: making it difficult to take a screenshot you mean?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9766b8079954aaec488672ffa95f8d7db71f7ce

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py (hunk 2 above)
Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py]: yes, by the time the screenshot is taken, it already exited the wait state.  and the duration fluctuates too, sometimes in less than 1/2 seconds, sometimes takes 10 seconds.

**Pull request merged**